GPU/GPU.cpp

```diff
@@ -39,8 +36,6 @@
 #include "GPU/D3D11/GPU_D3D11.h"
```

Comment:
Add a PPSSPP_API(D3D9) detection here maybe? It seems to break UWP compilation for now.

Reply:
Well I am talking about the line before it
the DirectX9 inlcude

Reply:
Done.